### ISSUE TITLE
Auto Archive Local Event Content

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#3172]
Split from https://github.com/uiowa/uiowa/issues/2396
Follow-up to #3170

# Story

As a site owner when creating local events with my local event content type feature split turned on, I can select to have the event auto-archive after the event passes.

# Proposed solution

### Auto Archive

- [ ] "Archive after event concludes" type boolean on content type.
- [ ]  Periodic cron job looks for past events and changes moderation to archive
- [ ] This functionality needs to be feature split aware and shouldn't run if the split is not enabled. - This wouldn't be necessary if sitenow_events is dedicated only to this functionality and not ui events - https://github.com/uiowa/uiowa/issues/3458

**Comment by joewhitsitt** [comment]
A nice way to do this would be to leverage a view to check for results. This is how we "auto close" alerts on the ITS website. Otherwise an entity query works as well.

**Comment by joewhitsitt** [comment]
@richardbporter would you say this is blocked by https://github.com/uiowa/uiowa/pull/3383 or do you think a poor cron task can run regularly for this?

**Comment by richardbporter** [comment]
It should run once a day at least which seems like it'd be enough, right? This doesn't seem as critical to be down to the minute like publish/unpublish.

**Comment by joewhitsitt** [comment]
Correct. I imagine someone will be impatient depending on the event, but they can manually archive them.

**Comment by briand44** [comment]
Check with Admissions, if they don't need this we can close and revisit if another customer is looking for this.
